specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -8514,7 +8514,7 @@
         },
         "requireGuestProvisionSignal": {
           "type": "boolean",
-          "description": "Specifies whether the guest provision signal is required to infer provision success of the virtual machine."
+          "description": "Specifies whether the guest provision signal is required to infer provision success of the virtual machine.  **Note: This property is for private testing only, and all customers must not set the property to false.**"
```

Comment:
Can the customer set this to true or not set it at all? If this parameter has to be set to a value, seems like this should say 'all customers must set this property to true.'

Reply:
Customer should set it to true or should not give requireGuestProvisionSignal at all.   @Drewm3 can confirm.

Reply:
I think it would be more clear to say that all customers must set it to true when used, but it's not a blocking concern so I'll sign off.